### PR TITLE
Added more configurations to column navigation for security

### PR DESCRIPTION
**Pull request by danrot**
Required for https://github.com/sulu-io/sulu/pull/1584

__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes
- [ ] submit changes to the documentation

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: husky_components/column-navigation/main.js [hunk 1]
```diff
@@ -129,15 +129,15 @@ define([], function() {
             optionsContainer: ['<div class="' + constants.optionsClass + ' grid-row"></div>'].join(''),
 
             optionsAdd: ['<div class="align-center add pointer">',
```
Comment (wachterjohannes) [inline husky_components/column-navigation/main.js]: format this block like:

```
optionsAdd: [
    '<div class="align-center add pointer">',
    '    <span class="fa-plus-circle"></span>',
    '</div>'
].join(''),

File: husky_components/column-navigation/main.js [hunk 2]
```diff
@@ -129,15 +129,15 @@ define([], function() {
...
-                '<span class="fa-plus-circle"></span>',
+                '    <span class="fa-plus-circle"></span>',
                 '</div>'].join(''),
 
             optionsSettings: ['<div class="align-center settings pointer drop-down-trigger">',
-                '   <span class="fa-gear inline-block"></span><span class="dropdown-toggle inline-block"></span>',
+                '    <span class="fa-gear inline-block"></span><span class="dropdown-toggle inline-block"></span>',
```
Comment (wachterjohannes) [inline husky_components/column-navigation/main.js]: same here

File: husky_components/column-navigation/main.js [hunk 3]
```diff
@@ -129,15 +129,15 @@ define([], function() {
...
                 '</div>'].join(''),
 
             optionsOk: ['<div class="align-center ok pointer">',
-                '   <span class="fa-check"></span>',
+                '    <span class="fa-check"></span>',
```
Comment (wachterjohannes) [inline husky_components/column-navigation/main.js]: and here

File: husky_components/column-navigation/main.js [hunk 4]
```diff
@@ -520,7 +594,7 @@ define([], function() {
             // if there is nothing loaded yet, create a "root"-column
             if (columnNumber === 0) {
                 this.columns[0] = {};
```
Comment (wachterjohannes) [inline husky_components/column-navigation/main.js]: i think you dont need this line

**Comment by danrot** [comment]
@wachterjohannes fixed

**Pull request merged**